OktaIdxExample/OktaIdxExample/Signin/TableView Cells/IDXSeparatorTableViewCell.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  IDXSeparatorTableViewCell.swift
+//  OktaIdxExample
+//
+//  Created by Mike Nachbaur on 2021-03-19.
```

Comment:
Please, update the header here and in `IDXTitleTableViewCell` :)

Reply:
Hah! Thanks, I thought I got them all. That reminds me that I should add a default comment header template in the Xcode project.